### PR TITLE
Fix #902: Highfi OptionsActivity

### LINKED ISSUE #902: Highfi OptionsActivity
Mock:
https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/2776cbb1-22b0-42c9-b70b-2136bdb361da/OP-Options-Menu-/
https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/7d442317-dd5a-40e7-adbd-25285d010874/OP-Options-Menu-

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #902 

## Mocks
https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/2776cbb1-22b0-42c9-b70b-2136bdb361da/OP-Options-Menu-/
https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/7d442317-dd5a-40e7-adbd-25285d010874/OP-Options-Menu-

## Screenshot
Screenshot_1585518383
Screenshot_1585518386

## Known Issues:
1. The border around the items is getting overlapped at the same time it is inconsistent with [ProfileList UI](https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/ddc3dadf-10e4-45f6-aeea-eb8a330e00ff/Settings-1-Profile-1/).  So kept it similar to `ProfileList` UI and will be followed in other UIs also.
2. The landscape mock does not provide full control over mock to check the padding/margin and therefore the padding margin in that has been kept similar to that of `HelpItems`.
3. Normally we keep bottom padding/margin in recyclerview but here it does not make sense because scrolling the list in landscape mode does not make sense when all items are visible. Therefore removed paddingBottom from recyclerview.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-land/option_app_language.xml [hunk 1]
```diff
@@ -13,15 +13,19 @@
...
+    android:paddingBottom="20dp">
 
     <TextView
       android:id="@+id/app_language_label_text_view"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:fontFamily="sans-serif"
-      android:padding="8dp"
       android:text="@string/app_language"
       android:textColor="@color/oppiaPrimaryText"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/option_app_language.xml]: In the mock textcolor is #000000DE.

File: app/src/main/res/layout-land/option_audio_language.xml [hunk 2]
```diff
@@ -13,15 +13,19 @@
...
+    android:paddingBottom="20dp">
 
     <TextView
       android:id="@+id/audio_language_label_text_view"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:fontFamily="sans-serif"
-      android:padding="8dp"
       android:text="@string/audio_language"
       android:textColor="@color/oppiaPrimaryText"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/option_audio_language.xml]: Ditto here and everywhere else.

**Review by veena14cs — APPROVED** [review]
LGTM beside one change.

File: app/src/main/res/layout-land/option_app_language.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-land/option_app_language.xml]: Done in 3 landscape and 3 portrait files.

File: app/src/main/res/layout-land/option_audio_language.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/layout-land/option_audio_language.xml]: Done in 3 landscape and 3 portrait files.

**Pull request merged**